hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/block/SCMDeletedBlockTransactionStatusManager.java

```diff
@@ -300,15 +302,15 @@ private void updateStatus(UUID dnId, long scmCmdId,
         }
         break;
       default:
-        LOG.error("Can not update to Unknown new Status: {}", newStatus);
+        LOG.error("Unable to update from unknown DN report status: {}", newStatus);
```

Comment:
Thanks for the patch @xichen01. Shouldn't the log here say `Unable to update to <status>` instead of `Unable to update from <status>`?
Even in the above log changes for `updateStatus()` it's not really clear if `DN report status` is the new status that the SCM command should be updated to. May I ask what was unclear in the old log messages?

Reply:
This has been updated.